src/test/java/sorinfoca/Tests/Tema8/pages/JavaScriptRedirectPage.java

```diff
@@ -0,0 +1,31 @@
+package sorinfoca.Tests.Tema8.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.util.Set;
+
+public class JavaScriptRedirectPage {
+
+    private WebDriver driver;
+
+    public JavaScriptRedirectPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void get() {
```

Comment:
Please rename to goToPage() or something that suggests this action of navigating to url. In this way it will be easier to understand what this method do and not confusing with a getter method. Thanks!